tests/ConstructorTest.php

```diff
@@ -28,4 +28,15 @@ public function testUnicodeOutput(): void
         $output = new Output\UnicodeOutput();
         $this->assertStringContainsString('♙ ', $output->render($chess));
     }
+
+    public function testReversedeOutput(): void
+    {
+        $chess = new Chess();
+        $output = new Output\AsciiOutput();
+        $this->assertStringStartsWith('   +---+---+---+---+---+---+---+---+
+ 8', $output->render($chess));
+        $chess->board->reverse();
+        $this->assertStringStartsWith('   +---+---+---+---+---+---+---+---+
+ 1', $output->render($chess));
+    }
```

Comment:
I had the idea that we could make a gif from a cli console. If wee add the auto gameplay mechanism to it, it would look nice.